activerecord/lib/active_record/migration/compatibility.rb

```diff
@@ -16,6 +16,13 @@ def self.find(version)
       V6_1 = Current
 
       class V6_0 < V6_1
+        def method_missing(method, *arguments, &block)
```

Comment:
I'm not sure if we should be more explicit here and list all the `SchemaStatments` method that we're trying to override.